src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
```

Comment:
The property 'spring.application.name' is missing. It is recommended to add this property back as it is used to identify the application in a distributed system.